opensrp-reveal/src/main/java/org/smartregister/reveal/presenter/TaskRegisterFragmentPresenter.java

```diff
@@ -62,6 +68,12 @@
 
     private boolean isActionClicked = true;
 
+    private TaskFilterParams filterParams;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'filterParams' could be replaced by a local variable.](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=4684568)